test/spec/app_spec.rb

```diff
@@ -1038,11 +1018,32 @@
       expect(@driver.page_source.include?('Auto-Refresh')).to be(true)
     end
 
-    # it "should cancel request through switch teams" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should give 404 when trying to acess a media that is not related to the project on the URL", bin1: true do
+      data = api_create_team_project_and_link 'https://twitter.com/TheWho/status/890135323216367616'
+      @driver.navigate.to @config['self_url'] + '/check/me'     
+      wait_for_selector("team__edit-button", :class) 
+      el = wait_for_selector("//span[contains(text(), 'Create Team')]",:xpath)
+      el.click
+      team = "testteam#{Time.now.to_i}"
+      el = wait_for_selector("create-team__team-display-name-input",:class)
+      el.click
+
+      fill_field('team-name-container', team, :id)
+      el = wait_for_selector('team-slug-container',:id)
+      el.click
+      el.clear
+      fill_field('team-slug-container', team, :id)
+      el = wait_for_selector("create-team__submit-button",:class)
+      el.click
+      wait_for_selector("team-members__profile-link",:class)
```

Comment:
From line 1023 until here: why do you need to create a team using the web UI?

Reply:
It is not necessary.
Updated to `      api_create_team(team: team)`

Reply:
Updated to `api_create_team_and_project()` indeed